packages/amplication-client/src/Entity/EntityFieldListItem.scss

```diff
@@ -8,17 +8,32 @@
     @include flexFullRowWithSpacing;
     margin-bottom: var(--default-spacing);
 
+    &:nth-child(2) {
+      padding-bottom: 12px;
+      border-bottom: 1px solid var(--black10);
+    }
+
     &:last-child {
       margin-bottom: 0;
     }
   }
   &__title {
     @include body1;
     color: var(--black100);
+    font-size: 14px;
```

Comment:
add include title that have font size and weight

Reply:
Found a better title with font-size as 14px✅